src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -812,13 +812,13 @@ private static String processPageSourceForGodMode(String content) {
                 .replaceAll(Const.ParamsNames.REGKEY + "%3D([a-zA-Z0-9]){1,}\\%", Const.ParamsNames.REGKEY + "%3D\\${regkey\\.enc}\\%")
                 
                 // maintain order for the two below
-                // questionid
-                .replaceAll("value=\"([a-zA-Z0-9-_]){62,}\"","value=\"\\${question\\.id}\"")
                 // regkey in unreg student page
-                .replaceAll("value=\"([a-zA-Z0-9-_]){50,}\"","value=\"\\${regkey\\.enc}\"")
+                .replaceAll("(type=\"hidden\" ?|name=\""+Const.ParamsNames.REGKEY+"\" ?|value=\"([a-zA-Z0-9-_]){1,}\" ?){3}","name=\""+Const.ParamsNames.REGKEY+"\" type=\"hidden\" value=\"\\${regkey\\.enc}\"")
```

Comment:
1. Godmode runs on the html BEFORE it is standardized. This was done because the standardization process used to sanitize the html and so we decided to run Godmode on the html first before the standardization. 

2. Now that the context is very tight, I though it would be alright to loosen the requirement. I guess 50 is good, but is there a lower number that would be more robust?